core/src/main/resources/hudson/PluginManager/table.jelly

```diff
@@ -58,6 +58,22 @@ THE SOFTWARE.
       <form method="post" action="install">
         <j:set var="isUpdates" value="${attrs.page=='updates'}" />
         <local:tabBar page="${attrs.page}" xmlns:local="/hudson/PluginManager" />
+
+        <div id="top-sticker" >
```

Comment:
I claim that this sticker movement indicates that the code within should be its own file so that one can move the position w/o having to copy the code around...